.ci/metrics/metrics.py

```diff
@@ -139,12 +199,25 @@ def upload_metrics(workflow_metrics, metrics_userid, api_key):
       metrics_userid: The userid to use for the upload.
       api_key: The API key to use for the upload.
     """
+
+    if len(workflow_metrics) == 0:
+        print("No metrics found to upload.", file=sys.stdout)
+        return
```

Comment:
Moved this here: I want to keep this check to avoid sending bad requests if we drop metrics in the future, but it wasn't ideal to keep it in the main loop anymore since we could see we always add metrics above.

Reply:
Makes sense to me.